Makefile

```diff
@@ -35,25 +35,38 @@ help:
 list:
 	@echo ${CITIES}
 
+$(filter %,$(CITIES)): %: \
+  ${DATA_DIR}/%.bbox \
+  ${DATA_DIR}/%.osm.pbf \
+  ${DATA_DIR}/%.gtfs.tar \
+  ${DATA_DIR}/%.nominatim.sql ${DATA_DIR}/%.nominatim_tokenizer.tgz \
+  ${DATA_DIR}/%.photon.tgz \
+  ${DATA_DIR}/%.mbtiles \
+  ${DATA_DIR}/%.graph.obj \
+  ${DATA_DIR}/%.valhalla.tar \
+  tag_images
+	@echo "Built $@"
+
+%.bbox:
+	@echo "Extracting bounding box for $(notdir $*)"
+	mkdir -p ${DATA_DIR}
+	rm -f ${DATA_DIR}/*.bbox
+	grep "$(notdir $*):" gtfs/bboxes.csv | cut -d : -f 2 > $@
+	cp $@ ${DATA_DIR}/bbox.txt
+
 %.osm.pbf:
 	mkdir -p ${DATA_DIR}
 	echo "Downloading $(notdir $*) from BBBike."
 	wget -U headway/1.0 -O $@ "https://download.bbbike.org/osm/bbbike/$(notdir $*)/$(notdir $@)" || rm $@
-	@echo "\n\nConsider donating to BBBike to help cover hosting! https://extract.bbbike.org/community.html\n\n"
+	@echo -e "\n\nConsider donating to BBBike to help cover hosting! https://extract.bbbike.org/community.html\n\n"
 
-%.bbox:
-	@echo "Extracting bounding box for $(notdir $*)"
-	grep "$(notdir $*):" gtfs/bboxes.csv > ${DATA_DIR}/bbox.txt
-	perl -i.bak -pe 's/$(notdir $*)://' ${DATA_DIR}/bbox.txt
-
-%.mbtiles: %.osm.pbf
-	@echo "Building MBTiles $*"
-	cp $*.osm.pbf mbtiles_build/data.osm.pbf
-	ITAG=headway_build_mbtiles_$$(echo $(notdir $*) | tr '[:upper:]' '[:lower:]')
-	docker build ./mbtiles_build --tag $${ITAG}
-	CID=$$(docker create $${ITAG})
-	docker cp $$CID:/data/output.mbtiles $@
-	docker rm -v $$CID
+%.gtfs.tar:
+	set -e ;\
```

Comment:
Looks like I missed to update this target. Since `.ONESHELL` and `SHELLFLAGS` were set, this line as well as the trailing ` ;\` on the rest of the lines in this block can be removed.